a-tier-bonding/src/routes/schedule/schedule.js

```diff
@@ -4,21 +4,21 @@ import CircularProgress from '@mui/material/CircularProgress';
 import "./schedule.css"
 
 function Schedule() {
-  const [loaded, setLoaded] = useState(false); 
-  const [schedule, setSchedule] = useState([]); 
+  const [loaded, setLoaded] = useState(false);
+  const [schedule, setSchedule] = useState([]);
 
   useEffect(() => {
     if (!loaded) {
       getGames().then(schedule => {
         setSchedule(schedule);
         setLoaded(true);
-      }) 
+      })
     }
   }, [loaded]);
   return (
-    loaded ? 
+    loaded ?
     <div className="schedule">
-      <h2>Schedule: All Times EST</h2>
+      <h2>Schedule: Times Shown in Local Timezone</h2>
```

Comment:
I only just realized this now, apparently it shows in local timezone LOL (edited this while in PST).